environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/handler/freeipa/FreeIpaCreationHandler.java

```diff
@@ -70,6 +74,8 @@
 
     private static final List<String> DEFAULT_SECURITY_GROUP_PORTS = List.of("22");
 
+    private static final String YARN_NETWORK_CIDR = "172.27.0.0/16";
```

Comment:
I think it should be an application property

Reply:
Done in af20f701d8c3e87addc57f612c9b6fc642d0daf5